src/app/TERMS.md

```diff
@@ -0,0 +1,252 @@
+
+We will not be liable for any loss or damage arising from your failure to protect your Account or your Wallets.
+
+It is your sole responsibility to provide accurate information for creation and maintenance of your account. You are solely responsible for ensuring the accuracy and completeness of all information and materials that you provide to use in connection with your use of the Site or Services. This includes, but is not limited to, all information and materials that you provide to us in connection with background checks and identity verification. You represent and warrant that (i) all such information and materials are true, accurate, and complete in all respects, comply with applicable law and do not violate or infringe any third-party rights; and (ii) you will immediately notify us about, and correct, and inaccuracy in any such materials or information.
+
+## Trading Terms
+
+We provide Users with a platform that allows orders they enter to be connected with the orders of other Users. Orders may be partially filled or may be filled by a number of orders, depending on the trading activity at the time an order is placed.
+
+A transaction on DeXter may fail for several reasons, including without limitation a change in prices, order availability, or technical difficulties experienced by us or Nodes. We make no representation or warrant that any transaction will be executed fully, or at all. We are, under no circumstances, liable for any loss or injury suffered by a failure of a transaction to complete properly or in a timely manner, including dispatching to the Radix Network. Further, we are in no way responsible for notifying you of a transaction failure, although you are able to see any such failures on the Site. You have full responsibility to determine and inquire into the failure of any transaction which you initiate.
+
+The digital assets that are available for purchase through the Services may be subject to high or low transaction volume, liquidity issues, and volatility for potentially extended periods. WE MAY ALSO DELIST AN ASSET FROM TRADING ON DeXter AT ANY TIME, FOR ANY REASON, WITHOUT PRIOR NOTICE. You acknowledge that while we are using commercially reasonable methods to provide trading availability and pricing data to you through our Services, we do not guarantee that the Services will be consistently available. You agree that you assume all risks and potential losses associated with digital assets being delisted, price fluctuations, or differences in actual versus indicated prices.
+
+We operate as a platform provider and generally do not act as principal or counterparty with respect to trades entered into on the platform. Notwithstanding the foregoing, (i) we may act as a counterparty for limited trades made for the purpose of liquidating fees collected on User trades, and (ii) our affiliates or partners may execute trades on the platform.
```

Comment:
I don't think the comment on liquidating fees on user trades is relevant to DeXter.